internal/reconcile/scc_test.go

```diff
@@ -36,11 +37,11 @@ var _ = Describe("reconciling ", func() {
 
 		globalScheme := scheme.Scheme
 		Expect(security.Install(globalScheme)).To(Succeed())
-		builder := fake.NewClientBuilder().WithScheme(globalScheme)
+
```

Comment:
If you want, you could also inline `globalScheme` above into the `Install()` call. It's just a fancy `AddToScheme` :slightly_smiling_face: